tests/components/samsungtv/test_media_player.py

```diff
@@ -348,11 +350,9 @@ def mock_now():
 async def test_turn_off_legacy(hass):
     """Test for turn_off."""
     with patch("homeassistant.components.samsungtv.config_flow.socket"), patch(
-        "homeassistant.components.samsungtv.config_flow.Remote",
+        "homeassistant.components.samsungtv.bridge.Remote",
         side_effect=[OSError("Boom"), mock.DEFAULT],
-    ), patch(
-        "homeassistant.components.samsungtv.media_player.SamsungRemote"
-    ) as remote_class, patch(
+    ), patch("homeassistant.components.samsungtv.bridge.Remote") as remote_class, patch(
```

Comment:
We don't need to patch the same path twice. Decide how it should be patched and do it once.